app/views/govuk_publishing_components/components/docs/button.yml

```diff
@@ -129,3 +129,13 @@ examples:
     data:
       text: Button
       aria_label: Button with custom label
+  with_aria_controls:
+    description: Used to identify what element(s) the button controls, this can be useful for users with visual impairment. In the example shown, the button controls a text area with and ID of `text-area-id` but it could be another identifying attribute.
+    data:
+      text: Button
+      aria_controls: text-area-id
```

Comment:
This value and the similar value for the `describedby` example are triggering unnecessary accessibility warnings in the component guide (see screenshot below). Could they be updated to actual values so that these warnings don't occur?

For example you could change `aria_controls` to `wrapper` (the id of the page wrapper) and `aria_describedby` to  `with_aria_describedby` (which is the id of that section).

![Screenshot 2022-11-29 at 09 40 31](https://user-images.githubusercontent.com/861310/204494222-f7a1862a-129b-4629-adbd-0f2cac811138.png)